pkg/alert/ctl_alert.go

```diff
@@ -89,27 +89,27 @@ func (ctl *HelmValuesFromCobraFlags) SetArgs(args map[string]interface{}) {
 // master=true is used to add all flags for creating an instance
 // master=false is used to add a subset of flags for updating an instance
 func (ctl *HelmValuesFromCobraFlags) AddCobraFlagsToCommand(cmd *cobra.Command, master bool) {
-	cmd.Flags().StringVar(&ctl.flagTree.Version, "version", ctl.flagTree.Version, "Version of Alert")
+	cmd.Flags().StringVar(&ctl.flagTree.Version, "version", "5.3.0", "Version of Alert")
 	if master {
 		cobra.MarkFlagRequired(cmd.Flags(), "version")
 	}
 
-	cmd.Flags().StringVar(&ctl.flagTree.StandAlone, "standalone", ctl.flagTree.StandAlone, "If true, Alert runs in standalone mode [true|false]")
+	cmd.Flags().StringVar(&ctl.flagTree.StandAlone, "standalone", "true", "If true, Alert runs in standalone mode [true|false]")
 	if master {
 		cmd.Flags().StringVar(&ctl.flagTree.ExposeService, "expose-ui", util.NONE, "Service type to expose Alert's user interface [NODEPORT|LOADBALANCER|OPENSHIFT|NONE]")
 	} else {
 		cmd.Flags().StringVar(&ctl.flagTree.ExposeService, "expose-ui", ctl.flagTree.ExposeService, "Service type to expose Alert's user interface [NODEPORT|LOADBALANCER|OPENSHIFT|NONE]")
 	}
 	cmd.Flags().StringVar(&ctl.flagTree.EncryptionPassword, "encryption-password", ctl.flagTree.EncryptionPassword, "Encryption Password for Alert")
 	cmd.Flags().StringVar(&ctl.flagTree.EncryptionGlobalSalt, "encryption-global-salt", ctl.flagTree.EncryptionGlobalSalt, "Encryption Global Salt for Alert")
-	cmd.Flags().StringVar(&ctl.flagTree.PersistentStorage, "persistent-storage", ctl.flagTree.PersistentStorage, "If true, Alert has persistent storage [true|false]")
+	cmd.Flags().StringVar(&ctl.flagTree.PersistentStorage, "persistent-storage", "false", "If true, Alert has persistent storage [true|false]")
```

Comment:
this need to be true right?

Reply:
This is false in the helm chart by default. Should we change the Helm Chart to be true?

Reply:
what is the default value in Operator @mphammer ?

Reply:
The default is "true". I think we need to change the Helm Chart if we want this change

Reply:
In Black Duck also, the default is true. Can you please change the Helm chart to true also? If it is difficult to merge the changes in Helm chart, at least we can make it true here.

Reply:
Yes, I will make the change in helm! :)